xrpl/models/transactions/transaction.py

```diff
@@ -256,7 +264,10 @@ def has_flag(self: Transaction, flag: int) -> bool:
         Returns:
             Whether the transaction has the given flag value set.
         """
-        return self.flags & flag != 0
+        if isinstance(self.flags, int):
+            return self.flags & flag != 0
+        else:  # is List[int]
+            return flag in self.flags
```

Comment:
don't need the `else` here. could also be expressed as:

```python
return (isinstance(self.flags, int) and self.flags & flag != 0) or flag in self.flags
```

Reply:
I think that's messier to read